tools/functional-tester/etcd-tester/failure.go

```diff
@@ -153,6 +153,10 @@ func (f *failureUntilSnapshot) Inject(c *cluster, round int) error {
 	retry := snapshotCount / 1000 * 3
 	for j := 0; j < retry; j++ {
 		lastRev, err = c.maxRev()
+		if err != nil {
```

Comment:
why do we need the continue statement?

if err is not nil and lastRev is 0, then the evaluation of line 162 is false. we will hit line 165 and back-off before retry. this is the expected behavior.